yarn/src/main/scala/org/apache/spark/deploy/yarn/AMDelegationTokenRenewer.scala

```diff
@@ -65,6 +65,8 @@ private[yarn] class AMDelegationTokenRenewer(
     sparkConf.getInt("spark.yarn.credentials.file.retention.days", 5)
   private val numFilesToKeep =
     sparkConf.getInt("spark.yarn.credentials.file.retention.count", 5)
+  private val discachedConfiguration =
```

Comment:
echoing @srowen's comment, can you rename this? I don't understand what `discached` means